third-culture-software/bhima
# Add abiliity to edit cost center allocation basis

Updated Cost Center editing to support selecting/changing the allocation basis.
Other minor cleanups and refactors.  Fixed previous misuses of 'allocation basis' instead of 'allocation method'.

Closes https://github.com/IMA-WorldHealth/bhima/issues/5896

**TESTING**
 - Verify that editing cost/fee center works for changing the step-down allocation method and allocation basis.
 - Verify that creating a new cost center works properly and prevents creation of cost centers with improper allocation methods or basis values.
 - Check the French translations for all these terms and suggest better translations!
